cmd/root.go

```diff
@@ -104,8 +116,40 @@ func NewRootCmd(ctx context.Context) *cobra.Command {
 		newCmdUninstall(),
 		newCmdDelete(config),
 		newCmdWatch(config),
-		version.NewVersionCommand(),
+		cliversion.NewVersionCommand(),
 	)
 
 	return cmd
 }
+
+func versionCheck(cmd *cobra.Command) {
```

Comment:
There are two `if`s on this function that should be inverted to reduce indentation.